mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_qm_holdings_and_srs_marc_holdings_create_job_profile.sql

```diff
@@ -0,0 +1,299 @@
+INSERT INTO ${myuniversity}_${mymodule}.job_profiles (id, jsonb) values
+('fa0262c7-5816-48d0-b9b3-7b7a862a5bc7', '{
+  "id": "fa0262c7-5816-48d0-b9b3-7b7a862a5bc7",
+  "name": "quickMARC Derive - Create Holdings and SRS MARC Holdings",
+  "description": "Load MARC Holdings and inventory holdings to create SRS MARC Holdings and Holdings",
```

Comment:
the same here "Load MARC Holdings to create SRS MARC Holdings and Inventory Holdings"